cli/src/cli.rs

```diff
@@ -1851,7 +1851,10 @@ mod tests {
             stake_account: bob_keypair.into(),
             staker: None,
             withdrawer: None,
-            lockup: Lockup { slot: 0, custodian },
+            lockup: Lockup {
+                epoch: 0,
```

Comment:
`epoch` doesn't mean much on its own. Should this be`num_epochs` or `epoch_height`?